src/UI/Components/ServiceInstanceForm/Helpers/createFormState.ts

```diff
@@ -93,9 +101,22 @@ export const createEditFormState = (
         case "Textarea":
         case "TextList":
         case "Text": {
-          acc[curr.name] = curr.type.includes("dict")
-            ? stringifyDict(originalAttributes?.[curr.name])
-            : cloneDeep(originalAttributes?.[curr.name]);
+          if (curr.type === "int?" || curr.type === "float?") {
+            const value = originalAttributes?.[curr.name];
+
+            //empty string assertion is for converting input form to JSON Editor
```

Comment:
This is not clear to me. 
The JSON editor is relying on the JSON-Schema. 
We cannot assign null like that there when the field is empty. 
I feel like I'm missing the actual problem.